services/credential-issuance-server/src/apis/credential.api.ts

```diff
@@ -87,9 +96,32 @@ async function issueCredentialWithKeriOobi(
   httpResponse(res, response);
 }
 
+async function applySchema(
+  req: Request,
+  res: Response
+): Promise<void> {
+  const { schemaSaid, aid } = req.body;
+  if (!SCHEMA_ACDC[schemaSaid]) {
```

Comment:
We should be able to request a disclosure of any ACDC, think we can remove this check